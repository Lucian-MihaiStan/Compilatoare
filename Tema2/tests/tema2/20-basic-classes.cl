class A inherits IO {
    a : Int <- self.copy().copy().copy();
};