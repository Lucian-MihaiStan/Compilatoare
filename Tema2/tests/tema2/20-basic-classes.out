"20-basic-classes.cl", line 2:16, Semantic error: Type Object of initialization expression of attribute x is incompatible with declared type Int
"20-basic-classes.cl", line 4:16, Semantic error: Type A of initialization expression of attribute a is incompatible with declared type Int
"20-basic-classes.cl", line 7:20, Semantic error: Type Int of initialization expression of attribute s2 is incompatible with declared type String
Compilation halted
